pfio/v2/fs.py

```diff
@@ -154,6 +155,11 @@ def is_forked(self):
         assert hasattr(self, 'pid')
         return self.pid != os.getpid()
 
+    @property
+    def is_trace(self):
```

Comment:
Perhaps `is_traced` is better grammatically; what do you think?

Reply:
Yes, I think so